examples/with-react-todos/src/todos/TodoList/events.js

```diff
@@ -12,7 +13,7 @@ export default function registerTodoListEvents() {
         },
       };
     },
-    [coeffect('global.get', 'apiUrl')]
+    [globalBuilder.get('apiUrl')]
```

Comment:
I'd prefer to read `global.get('apiUrl')`

Reply:
Problem here is that `global` is already a declared variable in Node, so I can not call it like that :(